astropy/table/tests/test_mixin.py

```diff
@@ -440,7 +440,13 @@ def init_from_class(c):
         for func in (copy.copy, copy.deepcopy, pickle_roundtrip, init_from_class):
             m2 = func(m)
             for attr in attrs:
-                assert getattr(m2.info, attr) == getattr(m.info, attr)
+                if (attr != 'dtype' or getattr(m.info.dtype, 'isnative', True) or
```

Comment:
Total nitpick: could you make this 3 lines like
```
if (attr != 'dtype'
        or getattr(m.info.dtype, 'isnative', True)
        or func in (copy.copy, copy.deepcopy)):
```
We've moved to having operators at the start of a line (and this would seem clearer).

Reply:
Thanks, I'd been scratching my head a bit around where we want to put the operators now (and the original version had a lot of `and not`s...), but could not quickly spot an example...